src/util/script_detection.js

```diff
@@ -263,6 +263,44 @@ export function charHasNeutralVerticalOrientation(char: number) {
  * @private
  */
 export function charHasRotatedVerticalOrientation(char: number) {
-    return !(charHasUprightVerticalOrientation(char) ||
-             charHasNeutralVerticalOrientation(char));
+    return !(exports.charHasUprightVerticalOrientation(char) ||
```

Comment:
`s/exports./` (x3)

Did this accidentally work because it gets transpiled to something where `exports.charInSupportedScript` actually works? Or are the tests incomplete?

Is there a lint we can turn on to flag the use of `exports`/`module.exports` in ES6 modules? (cc @anandthakker)

Reply:
Oops! 😅 I just ran the render tests in the debugger and can verify the functions are getting called. There's an "exports" object which appears to contain the exported functions:

```
 265 function charHasRotatedVerticalOrientation(char        ) {
>266     debugger;
 267     return !(exports.charHasUprightVerticalOrientation(char) ||
 268              exports.charHasNeutralVerticalOrientation(char));
debug> repl
Press Ctrl + C to leave debug repl
> exports
{ allowsIdeographicBreaking: undefined,
  allowsVerticalWritingMode: undefined,
  allowsLetterSpacing: undefined,
  charAllowsLetterSpacing: undefined,
  charAllowsIdeographicBreaking: undefined,
  ... }
> exports.charHasUprightVerticalOrientation
[Function: charHasUprightVerticalOrientation]
> exports.charHasUprightVerticalOrientation(char)
true
> exports.charHasNeutralVerticalOrientation(char)
false
> exports.charInSupportedScript(char)
true
```

Reply:
Hm, good point.  We should set `env.node` to `false` in our root `.eslintrc`.

Reply:
https://github.com/mapbox/mapbox-gl-js/issues/6422